src/components/Nav.tsx

```diff
@@ -1,14 +1,28 @@
 import { useEffect, useState, VFC } from "react"
-import { Container, Flex, FlexProps } from "@chakra-ui/react"
+import { Container, Flex, FlexProps, HStack } from "@chakra-ui/react"
 import { useConnect } from "wagmi"
 import ConnectButton from "components/_buttons/ConnectButton"
 import { Link } from "components/Link"
 import { LogoTextIcon } from "components/_icons"
+import { useRouter } from "next/router"
+
+export const NAV_LINKS = [
```

Comment:
I feel like we should abstract this to its own file. could live at `utils/navLinks.ts` or something along those lines